context/app/static/js/components/home-revision/AssayTypeBarChartContainer/utils.js

```diff
@@ -3,8 +3,12 @@ import produce from 'immer';
 
 function formatAssayData(assayData, colorKey) {
   const formattedData = assayData.aggregations.mapped_data_types.buckets.reduce((acc, d) => {
+    const snakeCaseDataType = d.key.mapped_data_type.replace(/ /g, '_');
+    // TODO get datasets to display from index
```

Comment:
```suggestion
    // TODO: Get datasets to display from index, instead of depending on patterns in names.
    // The first step is having a hierarchy for assay types:
    // https://github.com/hubmapconsortium/portal-ui/issues/1688
```

I like to reference the blocking issue in TODOs. It forces me to check that there is, in fact, an issue, and if someone is looking at this code in the future, it will give them the bigger picture so they can carry on the work.